script_automation/esa_ubuntu_k3_single/get_storage_key.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
is bash always under usr/bin/env? Isnt under /usr/bin?

Reply:
calling /usr/bin/env should find bash if it's not in /usr/bin.